dev/com.ibm.ws.app.manager/src/com/ibm/wsspi/application/handler/ApplicationInformation.java

```diff
@@ -94,4 +94,9 @@ public interface ApplicationInformation<T> {
      * @return
      */
     public boolean getUseJandex();
+
+    /**
+     * This indicates which libraries that are not usually considered war libs should be scanned for web application annotations
+     */
+    public String getAnnotationScanLibrary();
```

Comment:
```suggestion
    @Deprecated
    public String getAnnotationScanLibrary();
```
I believe new APIs are beta are supposed to start out being marked `@Deprecated` until they are moved to GA.